HW2_koshkareva/calculator.py

```diff
@@ -0,0 +1,59 @@
+# функция для расчёта суммы двух чисел
+def add(a, b):
+    return a + b
+
+# функция для расчёта разности двух чисел
+def subtract(a, b):
+    return a - b  
+  
+# функция для расчета частного двух чисел
+def divide(a, b):
+    if b == 0:
+        return "Внимание! Делитель равен 0! Замените делитель!"
+    else:
+        return a / b
+
+# функция для расчета произведения двух чисел
+def  multiply(a, b):
+	return a*b     
```

Comment:
операнды в математических операциях принято разделять запятыми
```suggestion
	return a * b     
```